docs/install/requirements.md

```diff
@@ -0,0 +1,51 @@
+RKE2 is very lightweight, but has some minimum requirements as outlined below.
+
+## Prerequisites
+
+Two nodes cannot have the same hostname.
+
+If all your nodes have the same hostname, use `--node-name` or `$RKE2_NODE_NAME` for each node you add to the cluster to have a different node name.
```

Comment:
i'm trying to make reference to config file params rather than cmd line args or env vars
```suggestion
If all your nodes have the same hostname, set the `node-name` parameter in the RKE2 config file for each node you add to the cluster to have a different node name.
```

Reply:
got it, will fix